microhapulator/api.py

```diff
@@ -40,19 +40,19 @@
 
 
 def count_and_sort(profile, include_discarded=True):
-    counts = dict(
-        Marker=list(),
-        ReadCount=list(),
-    )
+    counts = list()
     for marker, mdata in profile.data["markers"].items():
         readcount = 0
         if include_discarded:
             readcount += mdata["num_discarded_reads"]
         for haplotype, count in mdata["typing_result"].items():
             readcount += count
-        counts["Marker"].append(marker)
-        counts["ReadCount"].append(readcount)
-    data = pd.DataFrame(counts).sort_values(["ReadCount"], ascending=False).reset_index(drop=True)
+        counts.append((marker, readcount))
+    data = (
+        pd.DataFrame(counts, columns=["Marker", "ReadCount"])
+        .sort_values(["ReadCount", "Marker"], ascending=False)
+        .reset_index(drop=True)
+    )
```

Comment:
Made a few stylistic changes here as well, but the big change was adding "Marker" as a secondary sorting key. Prior to that, Python 3.9 on the Ubuntu VM (but not locally on macOS!) would sort values differently.